### DISCUSSION TITLE
React native support

### DISCUSSION THREAD

**Discussion by radekzz** [patternfly/patternfly#5447] [General]
Hey guys,
first of all I really appreciate your work. Now using [victory-native](https://formidable.com/open-source/victory/docs/native/) and I'm curious if you plan to support also RN.
I've tried some examples and they don't seem to be RN ready.

**Comment by nicolethoen** [comment]
At this point, there is no plan to support react native that i'm aware. Doesn't mean it couldn't be discussed in the future.
